### PR TITLE
Css values 5

### PR DESCRIPTION
**Pull request by vwallen**

### CODE REVIEW

File: features/attr.yml.dist [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "2"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "1"
+  #   firefox_android: "4"
+  #   safari: "3.1"
+  #   safari_ios: "2"
+  - css.types.attr
```
Comment (jamesnw) [inline features/attr.yml.dist]: I think I'd recommend splitting out the `attr.type-or-unit` into a separate feature, as that will be something people will refer to separately. It also would match https://caniuse.com/css3-attr, I think

Fallback could be a separate feature, or use compute_from for css.types.attr. I'm on the fence here.

File: features/calc-size.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+name: cacl-size
```
Comment (jamesnw) [inline features/calc-size.yml]:
```suggestion
name: calc-size
```

File: features/calc.yml [hunk 3]
```diff
@@ -3,5 +3,11 @@ description: The `calc()` CSS function computes mathematical expressions such a
 spec: https://drafts.csswg.org/css-values-3/#calc-notation
 group: css
 caniuse: calc
+status:
+  compute_from: css.types.calc
 compat_features:
   - css.types.calc
+  - css.types.calc.color_component
```
Comment (jamesnw) [inline features/calc.yml]: This one actually belongs in `relative-colors`, but is tagged in BCD incorrectly. I think remove it from here.

File: features/frequency.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+name: frequency
+description: The `frequency` CSS type represents a frequency value, expressed in hertz (`Hz`), or  kilohertz (`kHz`). The `frequency-percentage` CSS type represents a frequency or percentage values.
```
Comment (jamesnw) [inline features/frequency.yml]:
```suggestion
description: The `frequency` CSS type represents a frequency value, expressed in hertz (`Hz`), or kilohertz (`kHz`). The `frequency-percentage` CSS type represents a frequency or percentage values.
```

File: features/angle.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+name: angle
+description: The `angle` CSS type represents a portion of a circle, expressed in degrees (`deg`), gradians (`grad`), radians (`rad`), or turns (`turn`). The `angle-percentage` CSS type represents an angle or percentage value.
```
Comment (jamesnw) [inline features/angle.yml]: Do we need to mention the `angle-percentage` type? If it's just that a value can be either an angle or percentage, I think we just need to describe angle here, and the `angle | percentage` union type is only important in the scope of where it's used.

File: features/length.yml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+name: length
+description: The `length` CSS type represents a distance value. Distance values can be expressed in font-relative units such as ems (`em`), viewport-relative units such as viewport width (`vw`), or absolute units such as pixels (`px`). The `length-percentage` CSS type represents a distance or percentage value.
+spec: https://drafts.csswg.org/css-values-4/#frequency
```
Comment (jamesnw) [inline features/length.yml]: Wrong link

File: features/string.yml [hunk 7]
```diff
@@ -0,0 +1,7 @@
+name: string
```
Comment (jamesnw) [inline features/string.yml]: I think we'll likely need a disambiguation on this name and id. Probably on a few others as well.

File: features/angle.yml [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+spec: https://drafts.csswg.org/css-values-4/#angles
+group: css
```
Comment (jamesnw) [inline features/angle.yml]: Should we have a CSS types subgroup?

**Review by jamesnw — COMMENTED** [review]
I think overall this is in the right direction. A few comments throughout.

Could we also split this into 2 PRs? One would be the functions like `attr` or `calc` and then the other would be the types. I think there's a chance we'll want to have the types that have been around forever and are also things that you don't directly type be part of a base CSS feature with things like `css.selectors.class`. (I just discovered the selectors-4 draft by the way 😮 ).

File: features/angle.yml (hunk 8 above)
Comment (vwallen) [reply features/angle.yml]: It's definitely getting big enough. Would that work for typed OM as well?

File: features/angle.yml (hunk 5 above)
Comment (vwallen) [reply features/angle.yml]: If removed here, it should be removed from the related features as well. Is there any issue removing it an leaving the key?

**Pull request closed without merging**